MESC_Common/Src/MESCsin_lut.c

```diff
@@ -346,3 +352,92 @@ float sinwave[321] = {0,
                       0.9987954562,
                       0.9996988187,
                       1};
+void sin_cos_fast( int16_t angle , float * sin, float * cos)
+{
+	*sin = sinwave[angle >> 8];
+	*cos = sinwave[(angle >> 8) + 64];
```

Comment:
Strictly speaking, either `angle` should be `uint16_t` or `>> 8 `should be `/ 256`

Reply:
see new commit